### PR TITLE
Doc OIDC for edge native clusters

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR creates the PXK-E packs doc and documents OIDC options. It also fixes a couple of links in the PXK doc and adds a couple. 

## Review Changes

💻 [PXK-E](https://deploy-preview-1406--docs-spectrocloud.netlify.app/integrations/kubernetes-edge)
💻 [PXK](https://deploy-preview-1406--docs-spectrocloud.netlify.app/integrations/kubernetes)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PE-2076) and [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-646)

### CODE REVIEW

File: content/docs/06-integrations/00-kubernetes.md [hunk 1]
```diff
@@ -620,6 +622,7 @@ kubeadmconfig:
     oidc-extra-scope: profile,email,openid
 ```
 
+3. Provide third-party OIDC IDP details.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes.md:625]:
```suggestion
3. Provide third-party OIDC IDP details. Refer to the [SAML & SSO Setup](/user-management/saml-sso) for guidance on configuring a third party IDP with Palette.
```

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 2]
```diff
@@ -0,0 +1,1301 @@
...
+logoUrl: "https://registry.spectrocloud.com/v1/k8s-dashboard/blobs/sha256:2de5d88b2573af42d4cc269dff75744c4174ce47cbbeed5445e51a2edd8b7429?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Palette eXtended Kubernetes Edge
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:17]:
```suggestion
# Palette eXtended Kubernetes - Edge
```

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 3]
```diff
@@ -0,0 +1,1301 @@
+---
+title: "Palette eXtended Kubernetes Edge"
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:2]:
```suggestion
title: "Palette eXtended Kubernetes - Edge"
```

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 4]
```diff
@@ -0,0 +1,1301 @@
...
+metaTitle: "Palette eXtended Kubernetes Edge"
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:3]:
```suggestion
metaTitle: "Palette eXtended Kubernetes - Edge"
```

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 5]
```diff
@@ -0,0 +1,1301 @@
...
+metaDescription: "Learn about the Palette eXtended Kubernetes Edge pack and how you can use it your host clusters in an edge environment."
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:4]:
```suggestion
metaDescription: "Learn about the Palette eXtended Kubernetes - Edge pack and how you can use it your host clusters in an edge environment."
```

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 6]
```diff
@@ -0,0 +1,1301 @@
...
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Palette eXtended Kubernetes Edge
+
+The Palette eXtended Kubernetes (PXK-E) pack supports Kubernetes clusters set up on Edge hosts installed in isolated locations like grocery stores and restaurants versus a data center or cloud environment. We offer PXK-E as a core pack in Palette.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:19]:
```suggestion
The Palette eXtended Kubernetes - Edge (PXK-E) pack supports Kubernetes clusters set up on Edge hosts installed in isolated locations like grocery stores, restaurants, etc, versus a data center or cloud environment. We offer PXK-E as a core pack in Palette. 
```
The way it sounded was  Edge only goes in grocery stores and restaurants.

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 7]
```diff
@@ -0,0 +1,1301 @@
...
+# Versions Supported
+
+
+<Tabs>
+
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
+
+## Prerequisites
+
+- A minimum of 4 CPU and 4GB Memory.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:59]:
```suggestion
- A minimum of 2 CPU and 4GB Memory.
```
[Source](https://github.com/spectrocloud/pax/blob/0ca85f0a373e2b59f0fe6aca7628714e67b97ae4/stable/kubernetes/k8s_native_v1.25.2/pack.json#L48-L70)

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 8]
```diff
@@ -0,0 +1,1301 @@
...
+
+
+- Configured OpenID Connect (OIDC) Identity Provider (IDP).
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:62]:
```suggestion
```
Not a requirement.

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 9]
```diff
@@ -0,0 +1,1301 @@
...
+
+
+- Users or groups mapped to a Kubernetes RBAC role.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:65]:
```suggestion
```
Not a requirement

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 10]
```diff
@@ -0,0 +1,1301 @@
...
+## Usage 
+  
+The Kubeadm configuration file is where you can do the following:
+
+<br />
+
+- Change the default ``podSubnet`` and ``serviceSubnet`` values. CIDR IPs specified in the configuration file take precedence over other defined CIDR IPs in your environment.
+
+  As you build your cluster, check that the ``podSubnet`` value does not overlap with any hosts or with the service network and the ``serviceSubnet`` value does not overlap with any IP ranges assigned to nodes or pods. For more information, refer to the [Clusters](/clusters) guide and [Cluster Deployment Errors](https://docs.spectrocloud.com/troubleshooting/cluster-deployment). 
+
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:99]:
```suggestion
```
This I am not sure about, so let's take it out for now.

File: content/docs/06-integrations/00-kubernetes-edge.md [hunk 11]
```diff
@@ -0,0 +1,1301 @@
...
+- **None**: This setting does not require OIDC configuration for the cluster. It displays in the YAML file as `noauth`. 
+
+  <br />
+
+  <WarningBox>
+
+  We do not recommend choosing **None** in a production environment, as it may disable authentication for add-ons that rely on OIDC.
+
+  </WarningBox>
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kubernetes-edge.md:392]:
```suggestion
```

This warning does not feel right for Edge, as Edge are tiny environments with minimal footprint so organizations prefer to not make it easy to access these environments. Let's take it out.

**Review by karl-cardenas-coding — APPROVED** [review]
@ritawatson Minor suggestions. My feedback for PXK-E applies to all the tabs 😄

File: content/docs/06-integrations/00-kubernetes-edge.md (hunk 11 above)
Comment (ritawatson) [reply content/docs/06-integrations/00-kubernetes-edge.md:392]: You're right - this should come out.

**Pull request merged**